### ISSUE TITLE
Add ability to fetch CharacterID

### ISSUE THREAD

**Issue by mgibula** [eveseat/eseye#64]
Is there some publicly accessible API function for getting character ID from the token?

I can see that GuzzleFetcher already have all things required to read this from verifyToken(), but I can't find a way to access if without hacking the source.

**Comment by warlof** [comment]
Hi @mgibula 

Sounds a odd workflow, but from eseye you can do the following
`$eseye->getAuthentication()->access_token` - then, you can parse the string using JWT library and you'll get the character ID in it.

**Comment by mgibula** [comment]
Yeah, I also found out $esi->rawFetch() can be used to make custom call to /verify, which is also odd. 
I believe a public function to fetch this info would make live easier for newcomers:-)
